broker-cli/test/test-helper.js

```diff
@@ -9,7 +9,10 @@
 // unhandled promise rejections are not handled by default in mocha currently,
 // but are deprecated in node.js, so should indicate test failure
 // @see {@link https://github.com/mochajs/mocha/issues/2640|mochajs/mocha#2640}
-process.on('unhandledRejection', () => { throw new Error('Unhandled rejection during testing') })
+process.on('unhandledRejection', (err) => {
+  console.error(err)
```

Comment:
I don't think this error is the one that you want. Have you tested this?

Reply:
@treygriffith I had legit test failures that were being swallowed by this call. We should log the error JIC